integration_test/robots/login_robot.dart

```diff
@@ -228,6 +233,8 @@ class LoginRobot extends CoreRobot {
         getSignInBtn(),
```

Comment:
What happens if we click on signIn button success? How we can break the while loop?